ros2pkg/ros2pkg/verb/create.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import getpass
+import os
+import subprocess
+
+from ros2pkg.api.create import create_folder
+from ros2pkg.api.create import create_template_file
+from ros2pkg.verb import VerbExtension
+
+
+class CreateVerb(VerbExtension):
+    """Create a new ROS2 package."""
+
+    def add_arguments(self, parser, cli_name):
+        parser.add_argument(
+            'package_name',
+            help='The package name')
+        parser.add_argument(
+            '--destination-directory',
+            default=os.curdir,
+            help='Directory where to create the package directory')
+        parser.add_argument(
+            '--build-type',
+            default='ament_cmake',
+            choices=['cmake', 'ament_cmake'],
+            help='The build type to process the package with')
+        parser.add_argument(
+            '--dependencies',
+            nargs='+',
+            default=[],
+            help='list of dependencies')
+        parser.add_argument(
+            '--maintainer-email',
+            nargs='?',
+            help='email address of the maintainer of this package'),
+        parser.add_argument(
+            '--maintainer-name',
+            nargs='?',
+            help='name of the maintainer of this package'),
+        parser.add_argument(
+            '--cpp-node-name',
+            nargs='?',
```

Comment:
Why is the node name optional? If I pass only `--cpp-node-name` no executable gets generated.